source/_integrations/tplink.markdown

```diff
@@ -25,7 +25,9 @@ ha_integration_type: integration
 
 The `tplink` integration allows you to control your [TP-Link Smart Home Devices](https://www.tp-link.com/kasa-smart/) such as plugs, power strips, wall switches and bulbs.
 
-You need to provision your newly purchased device to connect to your network before it can be added via the integration. This can be done either by using [kasa command-line tool](https://python-kasa.readthedocs.io/en/latest/cli.html#provisioning) or by adding it to the official Kasa app before trying to add them to Home Assistant. If you use the app, do not upgrade the firmware if it presents the option to avoid blocking the local access by potential firmware updates.
+TP-Link offers an [FAQ](https://www.tp-link.com/us/support/faq/2707/) how to provision your newly purchased devices. The devices won't connect to the TP-Link cloud and become locally controllable afterwards. If you already own devices which are connected to the cloud, you can convert them from TP-Link Cloud operated to local operation as well.
+
+In the past more TP-Link used to support local access but they removed this support. If your old device is on older firmware it can still be controlled locally by Home Assistant. In this case you need to provision your newly purchased device to connect to your network before it can be added via the integration. This can be done by using the [kasa command-line tool](https://python-kasa.readthedocs.io/en/latest/cli.html#provisioning).
```

Comment:
```suggestion
In the past, more TP-Link used to support local access but they removed this support. If your old device is on older firmware it can still be controlled locally by Home Assistant. In this case, you need to provision your newly purchased device to connect to your network before it can be added via the integration. This can be done by using the [kasa command-line tool](https://python-kasa.readthedocs.io/en/latest/cli.html#provisioning).
```